src/Event.h

```diff
@@ -58,6 +58,13 @@ class EventMgr : public BroObj {
 	EventMgr();
 	~EventMgr() override;
 
+	void QueueEventFast(const EventHandlerPtr &h, val_list vl,
```

Comment:
Similar comment here - I think with the existence of QueueEventFast we now need documentation that explains the difference betwen QueueEvent & QueueEventFast exactly.

I know it is basically just an existence check - but without a comment like in the commit-message it is hard to get what is going on here.